src/schema/v2/discoverArtworks.ts

```diff
@@ -0,0 +1,44 @@
+import { GraphQLString, GraphQLFieldConfig } from "graphql"
+import { ResolverContext } from "types/graphql"
+import { artworkConnection } from "./artwork"
+import { connectionFromArray } from "graphql-relay"
+import { pageable } from "relay-cursor-paging"
+import gql from "lib/gql"
+
+export const DiscoverArtworks: GraphQLFieldConfig<void, ResolverContext> = {
+  type: artworkConnection.connectionType,
+  args: pageable({
+    userId: { type: GraphQLString },
+  }),
+  resolve: async (_root, args, { weaviateGraphqlLoader, artworksLoader }) => {
+    if (!weaviateGraphqlLoader) return
+    const { userId } = args
+
+    const query = gql`
+      query discoveryArtworksQuery {
+        {
+          Get {
+            DiscoveryArtworks(where: { userId: "${userId}" }) {
```

Comment:
Ok, so this is somewhat annoying, but the new class name is [`InfiniteDiscoveryArtworks`](https://github.com/artsy/quantum/blob/main/src/17-infinite-discovery/02-artworks.ts#L9). In our test weaviate instance, there is already a `DiscoveryArtworks`. 

Maybe this is fine, temporarily, but we'll need to make a decision around this. LMK what your thoughts are.

Reply:
No strong opinion on this, since it would involve offboarding the existing one and renaming the new one, right?